classes/Form/ConfigurationForm.php

```diff
@@ -218,6 +236,11 @@ public function treat()
             $treatmentResult .= $this->module->displayConfirmation($this->module->l('Account ID updated successfully'));
         }
 
+        if (null !== $gaV4Enabled) {
+            Configuration::updateValue('GA_V4_ENABLED', (bool) $gaV4Enabled);
+            $treatmentResult .= $this->module->displayConfirmation($this->module->l('Settings for Universal Analytics updated successfully'));
```

Comment:
Settings for `Google` Analytics `4` updated successfully

Reply:
Fixed